apps/src/storage/levelbuilder/ManifestEditor.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
+
+class ManifestEditor extends React.Component {
+  static propTypes = {
+    // Provided via Redux
+    libraryManifest: PropTypes.object.isRequired
+  };
+
+  render() {
+    return (
+      <div>
+        <h1>Edit Dataset Manifest </h1>
+        <textarea
+          id="content"
+          ref="content"
```

Comment:
Do you need both the id and ref? id should generally be avoided in React where possible.

Reply:
Fixed

Reply:
> id should generally be avoided in React where possible.

@jmkulwik i've never heard this before -- is there an article or something you can point me to on this?

Reply:
Hmm, good question. The reasoning I've heard is since React components are intended to be reusable, IDs shouldn't be used. i.e. to make the component reusable, you'd automatically have to refactor to remove IDs. When I searched "should I use IDs in React" I got a general consensus of "no" without a specific article. This was the closest to an article I found:
https://stackoverflow.com/questions/28519023/does-the-classname-attribute-take-on-the-role-of-the-id-attribute-in-reactjs
It's demonstrated here: https://www.javascriptstuff.com/use-refs-not-ids/
Granted, we often don't use react in a reusable way in our code. But if there was a duplicate id anywhere in the DOM tree, it could cause similar problems. If we do use an ID, it should probably be pretty specific to what we're doing.